providers/tests/standard/operators/test_latest_only_operator.py

```diff
@@ -82,6 +82,9 @@ def test_run(self, dag_maker):
         dag_maker.create_dagrun()
         task.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE)
 
+    @pytest.mark.skipif(
+        not AIRFLOW_V_3_0_PLUS, reason="execution_date is renamed to logical_date in Airflow 3.0"
+    )
```

Comment:
Actually... It would be better to not skip on > 3.0 but make the tests compatible with before 3.0

Reply:
Yeah, logical date has been there since ~2.4 so the test should be updated to not user execution_date.

We don't need execution_date at all anymore do we?

Reply:
Thanks, I have made the test conditional based on the version.

Reply:
nice!